Update _index.md (#27288)
DataDog/documentation: content/en/cloud_cost_management/datadog_costs/_index.md

You can view daily Datadog costs in [Cloud Cost Management][1], and access additional [Datadog costs capabilities][7] like [Cost Summary][9] and [Cost Chargebacks][10] on the [**Usage & Cost** page][4].

> Please note that there is **no additional charge** for Datadog Costs and is available for both CCM and non-CCM customers.

Reviewer: ```suggestion
There is **no additional charge** for Datadog Costs, and it is available for both CCM and non-CCM customers.
```
